src/status_im2/contexts/chat/messages/content/reactions/view.cljs

```diff
@@ -5,31 +5,79 @@
             [utils.re-frame :as rf]
             [status-im2.contexts.chat.messages.drawers.view :as drawers]))
 
+(defn- on-press
+  [own message-id emoji-id emoji-reaction-id]
+  (if own
+    (rf/dispatch [:models.reactions/send-emoji-reaction-retraction
+                  {:message-id        message-id
+                   :emoji-id          emoji-id
+                   :emoji-reaction-id emoji-reaction-id}])
+    (rf/dispatch [:models.reactions/send-emoji-reaction
+                  {:message-id message-id
+                   :emoji-id   emoji-id}])))
+
+(defn- on-long-press
+  [chat-id message-id emoji-id show-reaction-authors-sheet?]
+  (reset! show-reaction-authors-sheet? true)
+  (rf/dispatch [:chat.ui/emoji-reactions-by-message-id
+                {:message-id         message-id
+                 :chat-id            chat-id
+                 :long-pressed-emoji emoji-id}]))
+
+(defn message-reactions-row-comp
+  [{:keys [message-id chat-id]} user-message-content show-reaction-authors-sheet?]
+  (let [reactions                                 (rf/sub [:chats/message-reactions message-id chat-id])
+        [reaction-authors-list
+         reaction-authors-list-selected-reaction] (rf/sub [:message/reaction-authors-list chat-id])
+        _show-sheet                               (when (and reaction-authors-list
```

Comment:
why is it with `_` ? better to have defined function

Reply:
Undescore meaning it won't be used in the body of the let declaration.